client/discovery/discovery.js

```diff
@@ -0,0 +1,187 @@
+window.onload = function() {
+  if(typeof switch_is_present == 'undefined') mountDiscovery()
+}
+
+function mountDiscovery() {
+    "use strict"
+
+    var width,height
```

Comment:
vars... vars everywhere!
![giphy](https://user-images.githubusercontent.com/30317944/35769596-a67149a8-090d-11e8-8ff3-2bfdf62d18cc.gif)

Reply:
try to use const and let :smile: